### DISCUSSION TITLE
Dev priorities

### DISCUSSION THREAD

**Discussion by anonaddict** [FreeTubeApp/FreeTube#2599] [General]
I'd like to request more focus on QoL like not leaving fullscreen when you are watching a playlist, I probably have to return to fullscreen 30 or 40 times a day and it's a huge turn-off even after more than a year and it's pretty much the only thing stopping me from recommending freetube to others. I have no idea if this is hard to fix for some reason but it's pretty much the only bug I've run into for the last many months. I think you could gain a lot of users if there was more focus on   persistent settings overall. Currently it feels like you are actively fighting the app as it keeps resetting your various settings

In terms of priorities I'd put this ahead of even playlists 

Edit: since I can't add new comments on locked discussion I would like to clarify that it was never my intent to sound rude. I very much like freetube and the thread was intended to start a discussion on what people would like devs to focus on. Sorry if this is the wrong place for that. 

Again I very much respect and appreciate the effort you all put into making freetube

**Comment by absidue** [comment]
As mentioned in these two issues, this is not a simple feature to implement:
- https://github.com/FreeTubeApp/FreeTube/issues/2295
- https://github.com/FreeTubeApp/FreeTube/pull/1281

Asking for features to be implemented is fine but being rude and selfish about it is not. If you want to request features in the future, I recommend looking at the [open issues](https://github.com/FreeTubeApp/FreeTube/issues), to see how you can request features and report bugs politely and respectfully.

May I remind you that we are all volunteers, working on FreeTube in our spare time. If this is really such a big issue for you, feel free to open a pull request implementing it.
